mgr/stats: change in structure of perf_stats o/p (#46068)
ceph/ceph: doc/cephfs/cephfs-top.rst

Once enabled, Ceph Filesystem metrics can be fetched via::

  $ ceph fs perf stats

> Details of the JSON command output are as follows:

Reviewer: ```suggestion
The output format is json and contains fields as follows.
```
as done [here](https://docs.ceph.com/en/latest/cephfs/fs-volumes/#fs-subvolumes).

Author: Done!!

Once enabled, Ceph Filesystem metrics can be fetched via::

  $ ceph fs perf stats

>   {"version": 2, "global_counters": ["cap_hit", "read_latency", "write_latency", "metadata_latency", "dentry_lease", "opened_files", "pinned_icaps", "opened_inodes", "read_io_sizes", "write_io_sizes", "avg_read_latency", "stdev_read_latency", "avg_write_latency", "stdev_write_latency", "avg_metadata_latency", "stdev_metadata_latency"], "counters": [], "client_metadata": {"a": {"client.4152": {"IP": "192.168.1.7:0", "hostname": "fedora", "root": "/", "mount_point": "/mnt/cephfs", "valid_metrics": ["cap_hit", "read_latency", "write_latency", "metadata_latency", "dentry_lease", "opened_files", "pinned_icaps", "opened_inodes", "read_io_sizes", "write_io_sizes", "avg_read_latency", "stdev_read_latency", "avg_write_latency", "stdev_write_latency", "avg_metadata_latency", "stdev_metadata_latency"]}}}, "global_metrics": {"a": {"client.4152": [[5, 0], [0, 0], [0, 0], [0, 845888], [0, 0], [0, 1], [1, 1], [0, 1], [0, 0], [0, 0], [0, 0], [0, 0], [0, 0], [0, 0], [0, 845888], [0, 1]]}}, "metrics": {"delayed_ranks": [], "mds.0": {"client.4152": []}}}

Author: Yeah, as other `volume` commands o/p explanation is documented.But, this way it seems easier to look how the o/p is and see the explanation of fields.

>   {"version": 2, "global_counters": ["cap_hit", "read_latency", "write_latency", "metadata_latency", "dentry_lease", "opened_files", "pinned_icaps", "opened_inodes", "read_io_sizes", "write_io_sizes", "avg_read_latency", "stdev_read_latency", "avg_write_latency", "stdev_write_latency", "avg_metadata_latency", "stdev_metadata_latency"], "counters": [], "client_metadata": {"a": {"client.4152": {"IP": "192.168.1.7:0", "hostname": "fedora", "root": "/", "mount_point": "/mnt/cephfs", "valid_metrics": ["cap_hit", "read_latency", "write_latency", "metadata_latency", "dentry_lease", "opened_files", "pinned_icaps", "opened_inodes", "read_io_sizes", "write_io_sizes", "avg_read_latency", "stdev_read_latency", "avg_write_latency", "stdev_write_latency", "avg_metadata_latency", "stdev_metadata_latency"]}}}, "global_metrics": {"a": {"client.4152": [[5, 0], [0, 0], [0, 0], [0, 845888], [0, 0], [0, 1], [1, 1], [0, 1], [0, 0], [0, 0], [0, 0], [0, 0], [0, 0], [0, 0], [0, 845888], [0, 1]]}}, "metrics": {"delayed_ranks": [], "mds.0": {"client.4152": []}}}

Reviewer: @neesingh-rh 
I think you miss my point here. It's not making anything easier. rather it is difficult to maintain the docs and also misleading the user when the actual o/p is different.

Once enabled, Ceph Filesystem metrics can be fetched via::

  $ ceph fs perf stats

> The output format is json and contains fields as follows:

Reviewer 2: JSON

Author: Changed
